lib/fastifySession.js

```diff
@@ -102,6 +103,14 @@ function newSession (secret, request, cookieOpts, done) {
   done()
 }
 
+function destroySession (done) {
+  const request = this
+  request.sessionStore.destroy(request.session.sessionId, (err) => {
+    request.session = {}
```

Comment:
I think it is better to set the session to `null`.

Reply:
Changed successfully!